wiki/Registration/ko.md

```diff
@@ -0,0 +1,25 @@
+# 회원가입
+
+*경고: 하나이상의 osu!계정을 가지고 있는 것은 규칙 위반입니다! [osu! rules](/wiki/Rules)!* 참조
+
+1. osu! [설치](/wiki/Installation) 이후 실행하면, osu!는 로그인 할지 회원가입을 할지 물어봅니다.. 당신이 회원가입을 하기 위해 이글을 읽고잇다면, `Create an account`버튼을 클릭하세요.
+
+   ![The in-game registration form](img/ingame-registration.jpg "The registration form")
+
+2. 형식에 맞게 작성
+
+username 항목에는 알파벳 밑 숫자  (`a-z`, `A-Z`, `0-9`), 언더바 (`_`), 대괄호 (`[` and `]`), 대시키 (`-`), 스페이스바 키 (` `)를 포함한 최대 15글자 까지 가능합니다.
+    
+이메일 항목에는 당신의 이메일을 적으면 됩니다. 적은 이메일 주소는 비밀번호를 초기화하거나 본인확인을 해야할때 필요하니, 꼭 적어주세요!
+    
+비밀번호 항목에는 최대 8글자까지 가능합니다. 
+    

```

Comment:
```suggestion
   - username 항목에는 알파벳 밑 숫자  (`a-z`, `A-Z`, `0-9`), 언더바 (`_`), 대괄호 (`[` and `]`), 대시키 (`-`), 스페이스바 키 (` `)를 포함한 최대 15글자 까지 가능합니다.
   - 이메일 항목에는 당신의 이메일을 적으면 됩니다. 적은 이메일 주소는 비밀번호를 초기화하거나 본인확인을 해야할때 필요하니, 꼭 적어주세요!
   - 비밀번호 항목에는 최대 8글자까지 가능합니다. 
```

Reply:
why this code is error? I want to change that suggestion but It's an error